example/src/index.js

```diff
@@ -81,6 +181,11 @@ class App extends React.Component {
 		this.setState({
 			tree: tree
 		});
+		setTimeout(() => {
```

Comment:
pretty sure i was just experimenting with changing the max leaves.  a poor choice to leave it in there